easybuild/easyconfigs/s/SLATEC/SLATEC-4.1-slatec4linux-fPIC.patch

```diff
@@ -0,0 +1,9 @@
+diff -Nru slatec4linux-orig/static/makefile slatec4linux/static/makefile
```

Comment:
please add description/reason and author

Reply:
@migueldiascosta - done!  (After a rather lengthy delay - sorry!)

Reply:
@SimonPinches I'm afraid you'll also need to change the patch checksum :)